nc_time_axis/__init__.py

```diff
@@ -429,8 +431,11 @@ def convert(cls, value, unit, axis):
             value = value.reshape(-1)
             first_value = value[0]
         else:
-            # Don't do anything with numeric types.
-            if munits.ConversionInterface.is_numlike(value):
+            # Don't do anything with numeric types.  This check can be removed once
+            # the minimum version of matplotlib supported is at least 3.5, when
+            # convert is no longer required to support numeric or iterables of numeric
+            # types.  See GitHub issue 97 for more details.
```

Comment:
Convert to a `TODO:`, in line with commonly recognised syntax:

```python
    # TODO: remove this check once the minimum version of
    #  matplotlib supported is ...
```

Reply:
Good idea.  I went ahead and tagged the comments with TODO in all the places you suggested.